exchanges.yml

```diff
@@ -183,133 +183,118 @@ components:
           properties:
-            id:
-              type: string
-              description: The URL that uniquely identifies the created workflow.
-            steps:
+            stepName:
               type: object
-              description: One or more steps required to complete an exchange on the workflow.
+              description: The name of the step.
               properties:
-                stepName:
-                  type: object
-                  description: The name of the step.
-                  properties:
-                    step:
-                      $ref: "#/components/schemas/WorkflowStep"
-            initialStep:
-              type: string
-              description: The step from the above set that the exchange starts on.
-            controller:
-              type: string
-              description: The controller of the instance. Returning controller is OPTIONAL.
-            authorization:
-              type: object
-              description: Authorization scheme information (e.g., OAuth2 configuration). Returning authorization is OPTIONAL.
-            credentialTemplates:
-              type: array
-              description: One or more VC templates for issuance. Returning credentialTemplates is OPTIONAL.
-              items:
-                type: object
-                properties:
-                  type:
-                    type: string
-                    description: The type of template.
-                  template:
-                    type: string
-                    description: The template itself.
+                step:
+                  $ref: "#/components/schemas/WorkflowStep"
+        initialStep:
+          type: string
+          description: The step from the above set that the exchange starts on.
+        controller:
+          type: string
+          description: The controller of the instance. Returning controller is OPTIONAL.
+        authorization:
+          type: object
+          description: Authorization scheme information (e.g., OAuth2 configuration). Returning authorization is OPTIONAL.
+        credentialTemplates:
+          type: array
+          description: One or more VC templates for issuance. Returning credentialTemplates is OPTIONAL.
+          items:
+            type: object
+            properties:
+              type:
+                type: string
+                description: The type of template.
+              template:
+                type: string
+                description: The template itself.
     GetWorkflowResponse:
       type: object
       additionalProperties: false
       description: Object containing information about a workflow.
       properties:
-        config:
+        exchanges:
+          type: array
+          description: The identifiers of the current exchanges associated with the workflow instance.
+          items:
+            type: string
```

Comment:
This should be removed, returning all of the current exchanges isn't scalable -- and I don't think this is readily implemented either.